crates/transparency/src/log/vec_log.rs

```diff
@@ -67,6 +70,19 @@ where
 
         Some(result)
     }
+
+    /// Turn a VecLog into bytes using protobuf
+    pub fn encode(self) -> Vec<u8> {
```

Comment:
I would call these `to_/from_protobuf` for consistency with ProtoEnvelope.

Reply:
Cool with me. I'll rename.